wiki/component-development.md

```diff
@@ -62,18 +62,18 @@ yarn build && npm pack
 
 This will create a `.tgz` file with the changes in your OUI directory. At this point you can move it anywhere.
 
-#### In Kibana:
+#### In OpenSearch Dashboards:
 
-Point the `package.json` file in Kibana to that file: `"@opensearch-project/oui": "/path/to/elastic-oui-xx.x.x.tgz"`. Then run the following commands at Kibana's root folder:
+Point the `package.json` file in OpenSearch Dashboards to that file: `"@opensearch-project/oui": "/path/to/elastic-oui-xx.x.x.tgz"`. Then run the following commands at OpenSearch Dashboards root folder:
 
 ```bash
-yarn kbn bootstrap --no-validate && cd packages/kbn-ui-shared-deps/ && yarn kbn:bootstrap && cd ../../ && FORCE_DLL_CREATION=true node scripts/kibana --dev
+yarn osd bootstrap --no-validate && cd packages/osd-ui-shared-deps/ && yarn osd:bootstrap && cd ../../ && FORCE_DLL_CREATION=true node scripts/osd --dev
 ```
 
 * The `--no-validate` flag is required when bootstrapping with a `.tgz`.
   * Change the name of the `.tgz` after subsequent `yarn build` and `npm pack` steps (e.g., `elastic-oui-xx.x.x-1.tgz`, `elastic-oui-xx.x.x-2.tgz`). This is required for `yarn` to recognize new changes to the package.
```

Comment:
Should these references like `elastic-oui-xx.x.x-1.tgz` be changed to something like `oui-xx.x.x-1.tgz`?

Reply:
yea good catch